Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -159,23 +161,25 @@ final class CodecTests: MongoSwiftTestCase {
         expect(try encoder.encode(s1)).to(equal(doc1))
 
         // check that a UInt32 too large for an Int32 gets converted to Int64
-        expect(try encoder.encode(Numbers(uint32: 4294967295))).to(equal(["uint32": .int64(4294967295)]))
+        expect(try encoder.encode(Numbers(uint32: 4_294_967_295))).to(equal(["uint32": .int64(4_294_967_295)]))
 
         // check that UInt, UInt64 too large for an Int32 gets converted to Int64
-        expect(try encoder.encode(Numbers(uint64: 4294967295))).to(equal(["uint64": .int64(4294967295)]))
-        expect(try encoder.encode(Numbers(uint: 4294967295))).to(equal(["uint": .int64(4294967295)]))
+        expect(try encoder.encode(Numbers(uint64: 4_294_967_295))).to(equal(["uint64": .int64(4_294_967_295)]))
+        expect(try encoder.encode(Numbers(uint: 4_294_967_295))).to(equal(["uint": .int64(4_294_967_295)]))
```

Comment:
adding underscores in large number literals is another interesting change that's on by default. Let me know what you guys think about it.

Reply:
I like it 👍